MetaMask/metamask-extension
# Update links to docs in the README

Update the links to our docs in the "dapp developer resources" section

## **Description**
3 links in the README to the docs site are updated to the correct URLs. These links were using outdated URLs before.

## **Manual testing steps**

Click the links in the README to see that they go to the correct URLs now without any redirects

## **Pre-merge author checklist**

- [x] I’ve indicated what issue this PR is linked to: **N/A**
- [x] I’ve included tests if applicable. **N/A**
- [x] I’ve documented any added code. **N/A**

## **Pre-merge reviewer checklist**
